app/components/signatures/blockchain-status.js

```diff
@@ -1,6 +0,0 @@
-import Component from '@ember/component';
```

Comment:
The template file was [removed in a previous commit](https://github.com/lblod/frontend-gelinkt-notuleren/commit/ef9177a035f4ceaa88eacf3cca62c58290b375e3#diff-640d772fc45a12a869c2fdfaf71605911dd8ec442e266a1fb4c11b30db6969cb).